apitest.js

```diff
@@ -52,6 +52,10 @@ const callZAppAPIs = async (zappName , apiRequests, errorMessage, preHook) => {
         .request('localhost:3000')
         .post(apiRequests[i].endpoint)
         .send(apiRequests[i].data);
+      // Check if the response is successful
+      if (apiResponses[i].status !== 200) {
```

Comment:
We only check the status for the POST requests as the `/backupVariable` calls  (which are of kind GET) fail. It is fine as the requests for the Swap contract are all of kind POST so covered by this check.